spec/spec_helper.rb

```diff
@@ -1,3 +1,11 @@
+require 'capybara/rspec'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.